run.sh

```diff
@@ -21,7 +22,9 @@
 pip3 install pytest
 
 # Replace hyphen with underscore to follow Python snake-case filename convention
-test_file="${1/-/_}"
+test_file="${1//-/_}"
```

Comment:
This line is the functional change. All other changes were intended to validate the fix.